warehouse/accounts/models.py

```diff
@@ -111,6 +120,22 @@ class UnverifyReasons(enum.Enum):
     SoftBounce = "soft bounce"
 
 
+class TwoFactor(db.ModelBase):
+    __tablename__ = "two_factor"
+    __table_args__ = (
+        UniqueConstraint("user_id", name="two_factor_user_key"),
+        Index("two_factor_user_id", "user_id"),
+    )
+
+    id = Column(Integer, primary_key=True, nullable=False)
```

Comment:
Small nit, I know we use integer IDs elsewhere, but I generally prefer using UUIDs for a primary key. You can actually just delete this line, and inherit from ``db.Model`` instead to have the primary key taken care of for you automatically. We generally only use ``db.ModelBase`` for things that have primary keys that pre-date Warehouse.